addon/templates/components/nypr-accounts/membership-card/index.hbs

```diff
@@ -0,0 +1,156 @@
+{{#unless showPaymentHistory}}
+  {{#nypr-card classNames="nypr-membership-info" title="Membership Info" as |card|}}
+
+  {{#card.header as |header|}}
+    {{header.title text="Membership Info"}}
+  {{/card.header}}
+
+    {{!--
+    Possible user states to handle:
+    1. Active sustaining pledge (activeSustainingPledges)
+    2. Active one-time pledge, e.g. donated within last year (activeOneTimePledges)
+    3. Expired/declined sustaining pledge (isLapsedSustainer)
+    4. Expired/lapsed one-time pledge (isLapsedOneTimeMember)
+    5. Never pledged (isNonMember)
+    --}}
+
+    {{#if pledges.isPending}}
+      {{animated-loading-icon color="#CCC" classNames="membership-loading-animation"}}
+    {{/if}}
+    {{#if pledges.isFulfilled}}
+      <div class="pledge-order-info">
+        {{#if mostRecentSustainingPledgesPerOrderCode}}
+          <h2 class="active-donations-header">Active Recurring Donations</h2>
+          {{#each mostRecentSustainingPledgesPerOrderCode as |pledge|}}
+            <div class="pledge-container">
+              <div class="pledge-info-container">
+                <div class="pledge-fund-name">
+                  <div class="pledge-fund">{{pledge.fund}}</div>
+                </div>
+                <div class="pledge-info">
+                  <div class="pledge-order-price">${{pledge.orderPrice}}{{if pledge.isSustainer ' / month' }}</div>
+                  <div class="pledge-hyphen-separator">-</div>
+                  <div class="pledge-order-cc-type">{{pledge.creditCardType}} {{pledge.creditCardLast4Digits}}</div>
+                </div>
+              </div>
+              <div class="pledge-update-link">
+                <a href="https://pledge3.wnyc.org/donate/membership-sustainer/sustainer/sustainer?order_id={{pledge.orderKey}}">Update</a>
+              </div>
+            </div>
```

Comment:
I'd like to see this as a nested component; it'll help clean up this very long template